include/mbedtls/ecp.h

```diff
@@ -197,6 +197,27 @@ mbedtls_ecp_point;
  * odd prime as mbedtls_ecp_mul() requires an odd number, and
  * mbedtls_ecdsa_sign() requires that it is prime for blinding purposes.
  *
```

Comment:
Oh, since this has changed from just documentation to a new function, and [this resolves an issue that was just reported today](https://github.com/Mbed-TLS/mbedtls/issues/8045), can you please [add a changelog entry](https://github.com/Mbed-TLS/mbedtls/blob/development/ChangeLog.d/00README.md)? Something like:

```
Features
   * The documentation of mbedtls_ecp_group now describes the optimized
     representation of A for some curves. Fixes #8045.
```
(Not sure if Features is the right section?)